src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/adapters/ColumnListAdapter.js

```diff
@@ -113,23 +113,23 @@ class ColumnListAdapter extends AbstractAdapter {
         const {
             _permissions: {
                 view: viewPermission = true,
+                edit: editPermission = true,
             } = {},
         } = item;
 
         if (onItemClick) {
+            let itemIcon = 'su-eye';
```

Comment:
Personally I still try to avoid `let` :see_no_evil: I think I would even have preferred a nested ternary. I think they read quite nice, if you nest them correctly (and I've just checked, eslint doesn't even allow to nest them in a different way):

```javascript
const itemIcon = isGhost
    ? 'su-plus-circle'
    : editPermission
        ? 'su-pen'
        : 'su-eye';
```

They are also [gaining popularity in JS](https://medium.com/javascript-scene/nested-ternaries-are-great-361bddd0f340). Just a small input, I am also fine if we leave it this way, but I would slightly prefer the other way.

Reply:
sounds good to me 🙂